src/components/graph/graph.builder.js

```diff
@@ -155,11 +156,16 @@ function buildNodeProps(node, config, nodeCallbacks = {}, highlightedNode, highl
         label = config.node.labelProperty(node);
     }
 
+    let strokeWidth = node.strokeWidth || config.node.strokeWidth;
+
+    if (highlight && config.node.highlightStrokeWidth !== CONST.KEYWORDS.SAME) {
+        strokeWidth = config.node.highlightStrokeWidth;
+    }
+
     const t = 1 / transform;
     const nodeSize = node.size || config.node.size;
     const fontSize = highlight ? config.node.highlightFontSize : config.node.fontSize;
     const dx = fontSize * t + nodeSize / 100 + 1.5;
-    const strokeWidth = highlight ? config.node.highlightStrokeWidth : config.node.strokeWidth;
```

Comment:
Nice cleaning here 👍 Now we can use the `SAME` keyboard for `strokeWitdh` as well!

Reply:
Great! Do you want me to change the default value here
https://github.com/danielcaldas/react-d3-graph/blob/82474318f3d1e9aa4edcbf2d568b3d94e9ddb875/src/components/graph/graph.config.js#L222
and in the documentation?

Reply:
My newest commit includes this change.